Packages/MIES/MIES_MiesUtilities.ipf

```diff
@@ -5849,3 +5856,14 @@ Function NewExperiment()
 
 	Execute/P/Q "NEWEXPERIMENT "
 End
+
+/// @brief Remove the volatile part of the XOP error code
```

Comment:
@t-b what does volatile mean in this context?

Reply:
@timjarsky It should mean something like "moving"/"flexibel"/"non-constant". Does that make sense?